userguide/snippets/es60.rst

```diff
@@ -191,7 +191,11 @@ the top cover. Slide the top cover forward, then lift it off.
 Install Drive Trays
 ^^^^^^^^^^^^^^^^^^^
 
-Drive trays are used to mount drives in the array.
+Drive trays are used to mount drives in the array. These trays have
+two status lights for drives: green and amber. The drive tray light is
+solid green when a drive is connected, solid amber if there is an issue
```

Comment:
Why the separate mention of "issue" and "fault"?  Is there a difference?  Can we remove "issue"?